test/smoketest/main.py

```diff
@@ -516,6 +516,16 @@ def county_audit(ac, county_id):
     for i in range(len(selected)):
         if i % 10 == 0:
             r = test_endpoint_get(ac, ac.state_s, "/dos-dashboard", show=False)
+            
+        if i % 40 == 0:
```

Comment:
This should not happen as the first step. Perhaps after 5, to make it take effect in the default smoketest.

if i % 40 == 5:

Reply:
Fair enough. That's an easy change. In reality it doesn't actually matter when it occurs, I'm just checking to see that the logic works.